codegen/java-gen/src/it/scala/com/lightbend/akkasls/codegen/java/JavaCodegenIntegrationSuite.scala

```diff
@@ -1,582 +0,0 @@
-            println(line)
-            writer.write(s"${line}\n")
-          }
-      )
-    }
-    codegenContainer.copyFileToContainer(MountableFile.forHostPath(implFile), implContainerPath)
-
-    // Generate and compile the entity
-    assertSuccessful(
-      codegenContainer.execInContainer(
-        "./scripts/setup-entity.sh",
-        entityName
-      )
-    )
-  
-    // Start the entity gRPC server
-    assertSuccessful(
-      codegenContainer.execInContainer(
-        "./scripts/start-entity.sh",
-        entityName
-      )
-    )
-
-    // Start the proxy
-    proxyContainer.start()
-    val proxyUrl = s"http://${proxyContainer.getHost}:${proxyContainer.getMappedPort(9000)}"
-
-    // Eventually, a SingleMethod request should return a 200 OK
-    var getResult = retryUntil[requests.Response](_.statusCode == 200) {
-      requests.post(
-        s"$proxyUrl/com.example.MyActionService/SingleMethod",
-        check = false,
-        data = """{"value": 99}""",
-        headers = Map("Content-Type" -> "application/json")
-      )
-    }
-    assertEquals(getResult.statusCode, 200)
-    assertEquals(getResult.text(), """{"value":100}""")
-
-
-    val streamedResult = requests.post(
-        s"$proxyUrl/com.example.MyActionService/StreamedMethod",
-        check = false,
-        data = """{"value": 3}""",
-        headers = Map("Content-Type" -> "application/json")
-      )
-    assertEquals(streamedResult.statusCode, 200)
-    
-    assertEquals(
-      streamedResult.text(), 
-      """|{"value":1}
-         |{"value":2}
-         |{"value":3}
-         |""".stripMargin
-    )
-    
-    // Kill the gRPC server, and stop the proxy
-    assertSuccessful(
-      codegenContainer.execInContainer(
-        "./scripts/stop-entity.sh",
-        entityName
-      )
-    )
-    proxyContainer.stop()
-  }
-
-
-  def generateEntity(
-      container: AkkaslsJavaCodegenContainer
-  )(
-      artifactId: String,
-      groupId: String = "com.example",
-      version: String = "1.0-SNAPSHOT"
-  ): Container.ExecResult =
-    container.execInContainer(
-      "mvn",
-      "-B",
-      "archetype:generate",
-      "-DarchetypeGroupId=com.lightbend",
-      "-DarchetypeArtifactId=maven-archetype-akkasls-event-sourced-entity",
-      "-DarchetypeVersion=1.0-SNAPSHOT",
-      s"-DgroupId=${groupId}",
-      s"-DartifactId=${artifactId}",
-      s"-Dversion=${version}"
-    )
-
-  def assertSuccessful(result: Container.ExecResult) = {
-    val success = result.getExitCode() == 0
-    if (!success) {
-      logger.error(result.getStdout())
-    }
-    assert(success, () => result.getStderr())
-  }
-
-  def retryUntil[A](condition: A => Boolean, limit: Integer = 20, delay: Long = 1000)(
-      function: => A
-  ): A =
-    Some(function).filter(condition).getOrElse {
-      if (limit > 0) {
-        Thread.sleep(delay)
-        retryUntil(condition, limit - 1, delay)(function)
-      } else throw new Exception("Retry limit reached")
-    }
-}
```

Comment:
I guess all this will be replaced with something based on #57 ?

Reply:
Three of the samples now are part of the CI-build with the built codegen, and at least two of those run integration tests with actual proxy, so perhaps a tricky containery test like this is no longer needed. Not quite sure.